docs/tutorials/python/dataset_collection.md

```diff
@@ -0,0 +1,112 @@
+# Dataset Collections
+Dataset Collections are a way to organize, annotate, and publish sets of datasets for others to use. Dataset Collections behave similarly to Tables and EntityViews, but provide some default behavior that makes it easy to put a group of datasets together.
+
+This tutorial will walk through basics of working with Dataset Collections using the Synapse Python Client.
+
+# Tutorial Purpose
+In this tutorial, you will:
+
+- Create a Dataset Collection
+- Add datasets to the collection
+- Add a custom column to the collection
+- Update the collection with new annotations
+- Query the collection
+- Save a snapshot of the collection
+
+# Prerequisites
+* This tutorial assumes that you have a project in Synapse and have already created datasets that you would like to add to a Dataset Collection.
+* If you need help creating datasets, you can refer to the [dataset tutorial](./dataset.md).
+* Pandas must be installed as shown in the [installation documentation](../installation.md)
+
+## 1. Get the ID of your Synapse project
+
+Let's get started by authenticating with Synapse and retrieving the ID of your project.
+
+```python
+{!docs/tutorials/python/tutorial_scripts/dataset_collection.py!lines=9-14}
```

Comment:
This should probably also include the import lines of the script so one can copy/follow-along